notes/general/intro.md

```diff
@@ -0,0 +1,32 @@
+## Introduction
+
+### What is Algorithm?
+
+Informally, algorithms is well-defined sequence of steps that takes some values (input) and gives out some values after calculation (output).
```

Comment:
1) "algorithms is well-defined" > "algorithms are a well-defined"
2) "gives out" > "returns some"